openapi.yaml

```diff
@@ -0,0 +1,1861 @@
+openapi: 3.0.0
+info:
+  title: Substrate API Sidecar
+  description: REST API service intended to run next to Substrate, exposing a limited  set
+    of endpoints over HTTP with meaningful responses.
+  contact:
+    name: Parity Technologies
+    email: admin@parity.io
+  license:
+    name: GNU General Public License v3.0
+    url: https://github.com/paritytech/substrate-api-sidecar/blob/master/LICENSE
+  version: 1.0.0-oas3
+tags:
+- name: accounts
+- name: blocks
+- name: node
+  description: node connected to sidecar
+- name: pallets
+  description: pallets employed in the runtime
+- name: runtime
+- name: transaction
+paths:
+  /accounts/{accountId}/balance-info:
+    get:
+      tags:
+      - accounts
+      summary: Get balance information for an address.
+      description: Returns a curated grouping of values related to an accounts balance.
+        Replaces `/balance/{address}` from versions < v1.0.0.
+      operationId: getBalanceSummaryByAccountId
+      parameters:
+      - name: accountId
+        in: path
+        description: the account identifier, as the ss58 encoded public address.
+        required: true
+        style: simple
+        explode: false
```

Comment:
This can/should be omitted as it has no effect on string types. https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.2.md#parameterObject